core/include/irods/private/http_api/transport.hpp

```diff
@@ -0,0 +1,78 @@
+#ifndef IRODS_HTTP_API_TRANSPORT_HPP
+#define IRODS_HTTP_API_TRANSPORT_HPP
+
+#include <string_view>
+#include <memory>
```

Comment:
Let's move std headers after third-party headers.

This convention is used throughout this project and other iRODS projects.